src/NSubstitute.Analyzers.VisualBasic/DiagnosticAnalyzers/ReEntrantCallFinder.cs

```diff
@@ -46,6 +46,14 @@ public override void VisitInvocationExpression(InvocationExpressionSyntax node)
                 base.VisitInvocationExpression(node);
             }
 
+            public override void VisitClassBlock(ClassBlockSyntax node)
+            {
```

Comment:
@dtchepak The bug doesn't occur for VB analyzers - at least for the sample code reported in the ticket. However I decided to add this one and one below as a precaution. This will probably drop test coverage by a fraction of percentage.